src/bootstrap-table.js

```diff
@@ -976,6 +976,11 @@ class BootstrapTable {
       return value
     })
 
+    const paginationParts = opts.paginationParts
```

Comment:
`this.paginationParts = `

Reply:
Im working on it right now, check it out in ~5 minutes :-)